commands/daylist/daylist.js

```diff
@@ -39,106 +39,92 @@ function mergeAndUpdate(interaction, userId, split) {
   try {
     // Read existing content from JSON.
     readJsonFile((data) => {
-      const existing = data.users.find((user) => user[userId]) || {};
-      const existingAll = data.allWords;
-
-      // Track the new words being written for the user.
-      const userLog = {};
-      const newAllWords = {};
-
-      // Merge split with existing user words, updating counts if words already exist.
-      const existingUserWords = existing[userId] || {};
-      const userWords = split.reduce(
-        (result, word) => {
-          const lowercase = word.toLowerCase();
-          result[lowercase] = (result[lowercase] || 0) + 1;
-          userLog[lowercase] = result[lowercase]; // Track added words and their count.
-          return result;
-        },
-        { ...existingUserWords }
-      );
-
-      // Sort userWords by count in descending order
-      const sortedUserWords = Object.entries(userWords).sort(
-        (a, b) => b[1] - a[1]
-      );
-      const sortedUserWordsObject = Object.fromEntries(sortedUserWords);
-
-      // Track new words collectively for all users.
-      const allUserWords = new Set(
-        data.users.reduce((allWords, user) => {
-          const words = user[Object.keys(user)[0]];
-          return allWords.concat(Object.keys(words));
-        }, [])
-      );
-
-      const allWords = split.reduce(
-        (result, word) => {
-          const lowercase = word.toLowerCase();
-          if (!existingAll[lowercase]) {
-            if (!result[lowercase]) {
-              result[lowercase] = 0;
+      const usersData = data.users[userId] || {};
+
+      var globalUniqueWords = new Set();
+      var yourUniqueWords = new Set();
+
+      // Populate the sets of unique words
+      for (user in data.users) {
+        for (daylistObj of data.users[user]) {
```

Comment:
shouldnt we be using userData instead of data.users[user] here? What happens if the data is empty?

Reply:
i guess since this is in a try catch we wont need to worry about griddy crashing, so nothing would happen in this case.
But also, it means we won't know what happened since our console log aint great

Reply:
I think `usersData` might be like the "current" data

Reply:
wait, I am dumb 🤡 This code is fine

Reply:
I shall now delete this from all memory